public/app/features/provisioning/hooks/useIsProvisionedNG.ts

```diff
@@ -0,0 +1,20 @@
+import { useUrlParams } from 'app/core/navigation/hooks';
+
+import { DashboardScene } from '../../dashboard-scene/scene/DashboardScene';
+import { useGetFrontendSettingsQuery } from '../api';
+
+import { useGetResourceRepository } from './useGetResourceRepository';
+
+export function useIsProvisionedNG(dashboard: DashboardScene): boolean {
+  const [params] = useUrlParams();
+  const folderUid = params.get('folderUid') || undefined;
+
+  const folderRepository = useGetResourceRepository({ folderUid });
+  const { data } = useGetFrontendSettingsQuery();
+
+  return (
+    dashboard.isProvisioned() ||
+    Boolean(folderRepository) ||
+    Boolean(data?.items.some((item) => item.target === 'instance'))
```

Comment:
There'd probably be an easier way to check if the whole instance is provisioned.

Reply:
we used to have a flag in the settings -- but it essentially does the same you have here.  I don't know what is the best path